python/activity_stream/activity_stream.py

```diff
@@ -185,6 +186,24 @@ def _set_show_sg_stream_button(self, state):
         _get_show_sg_stream_button,
         _set_show_sg_stream_button,
     )
+
+    def _get_version_items_playable(self):
+        """
+        Whether the label representing a created Version entity is shown
+        as being "playable" within the UI. If True, then a play icon is
+        visible over the thumbnail image, and no icon overlay is shown
+        when False.
+        """
+        return self._version_items_playable
+
+    def _set_version_items_playable(self, state):
+        self._version_items_playable = bool(state)
+
+    version_items_playable = QtCore.Property(
+        bool,
+        _get_version_items_playable,
+        _set_version_items_playable,
+    )
```

Comment:
The getter is what's sourced by Sphinx, so that's where the docs should go for properties across the board. If we want additional docs for the setter for the sake of a dev reading the code that's fine, but it won't show up in Sphinx docs.